it/controllers/pensions/annualAllowance/AboveReducedAnnualAllowanceControllerISpec.scala

```diff
@@ -0,0 +1,333 @@
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          radioButtonCheck(yesText, 1, checked = Some(false))
+          radioButtonCheck(noText, 2, checked = Some(true))
+          buttonCheck(buttonText, continueButtonSelector)
+          textOnPageCheck(expectedDetails, detailsSelector)
+          formPostLinkCheck(aboveReducedAnnualAllowanceUrl(taxYearEOY), formSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+      }
+    }
+
+    //TODO redirect to annual CYA Page
+    "redirect to Pension Summary page if there is no session data" which {
+      lazy val result: WSResponse = {
+        dropPensionsDB()
+        authoriseAgentOrIndividual(isAgent = false)
+        urlGet(fullUrl(aboveReducedAnnualAllowanceUrl(taxYearEOY)), follow = false,
+          headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+      }
+
+      "has an SEE_OTHER status" in {
+        result.status shouldBe SEE_OTHER
+        result.header("location").contains(pensionSummaryUrl(taxYearEOY)) shouldBe true
+      }
+
+    }
+  }
+
+  ".submit" should {
+    userScenarios.foreach { user =>
+
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+
+        "return an error when form is submitted with no entry" which {
+          lazy val form: Map[String, String] = Map(YesNoForm.yesNo -> "")
+          lazy val result: WSResponse = {
+            dropPensionsDB()
+            val pensionsViewModel = aPensionAnnualAllowanceViewModel.copy(aboveAnnualAllowanceQuestion = None)
+            insertCyaData(pensionsUsersData(isPrior = false, aPensionsCYAModel.copy(pensionsAnnualAllowances = pensionsViewModel)), aUserRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlPost(fullUrl(aboveReducedAnnualAllowanceUrl(taxYearEOY)), body = form, welsh = user.isWelsh, follow = false,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          "has the correct status" in {
+            result.status shouldBe BAD_REQUEST
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedErrorTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          radioButtonCheck(yesText, 1, checked = Some(false))
+          radioButtonCheck(noText, 2, checked = Some(false))
+          buttonCheck(buttonText, continueButtonSelector)
+          textOnPageCheck(expectedDetails, detailsSelector)
```

Comment:
Can you add a Link check too for the external link
Example [here](https://github.com/hmrc/income-tax-pensions-frontend/blob/main/it/controllers/pensions/annualAllowance/ReducedAnnualAllowanceTypeControllersControllerISpec.scala#L199)